pygount/command.py

```diff
@@ -334,6 +336,9 @@ def execute(self):
 
         try:
             writer_class = _OUTPUT_FORMAT_TO_WRITER_CLASS_MAP[self.output_format]
+
+            if issubclass(writer_class, pygount.write.SummaryWriter):
```

Comment:
Is there any reason only the `SummaryWriter` would track the progress? To me it seems a progress bar would be nice for all writers.

Reply:
Only the summary writer displays the results at the end. I can tinker a little bit and add it for everything. The original reason I used this check is to leave your project completely untouched unless rich was used. So yeah, I will fix that.